### ISSUE TITLE
Queue Lost on App Closure

### ISSUE THREAD

**Issue by BOB74j** [z-huang/InnerTune#429]
### Checklist

- [X] I am able to reproduce the bug with the [latest DEBUG version](https://github.com/z-huang/InnerTune/actions).
- [X] I've checked that there is no issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

1. Set Persistent Queue to off in the InnerTune Player and audio settings.
2. Start playing a playlist in InnerTune.
3. Pause the playback.
4. Enter the recent apps screen and swipe to close InnerTune.
5. Re-open InnerTune. No music is queued.

### Expected behavior

In order to mimic the player behavior of the official YouTube Music app, the queue, the last known position in the queue, and the last known timestamp of the queued track should be retained across app sessions; regardless of the Persistent queue setting.

### Actual behavior

With Persistent queue off, after InnerTune closes or leaves RAM, the queue is gone. No mini-player is shown at the bottom of the app upon re-opening InnerTune. The previous position in an album or playlist must be recalled and selected manually.

Setting Persistent Queue to on retains the queue across app closures as expected. However, a side effect of this setting is a deleterious foreground service and a troubling tendency to respawn the player notification at unusual times, such as connecting to a VPN or turning on the system Bluetooth radio.

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/43751378/202012574-d93cae17-7dfc-450b-b0a8-70401138b397.mp4

### Music version

0.4.3

### Android version

Android 10

**Comment by z-huang** [comment]
> A side effect of Persistent queue on is a deleterious foreground service (whether music is playing or not)

This is wrong. No foreground service is running when you stop music. Persistent queue saves your queue only when the music stops playing.
